src/ChatWindow.test.js

```diff
@@ -160,15 +161,20 @@ it("handles the user's form submission with request to Alexa and populates the s
   const alexaId = chatterIds.ALEXA;
   const alexa = chatters.get(alexaId);
 
-  let expectedAlexaResponse = new Message({
-    id: alexaId,
-    message: mockAlexaSuccessResponse,
-    senderName: alexa.name
-  });
+  let expectedAlexaResponses = List();
+  for (let alexaResponse of mockAlexaSuccessResponses) {
+    expectedAlexaResponses = expectedAlexaResponses.push(
+      new Message({
+        id: alexaId,
+        message: alexaResponse,
+        senderName: alexa.name
+      })
+    );
+  }
```

Comment:
why not use a map() function here? Something like this.
```js
const expectedAlexaResponses = mockAlexaSuccessResponses.map(  
     (alexaResponse) => new Message({
        id: alexaId,
        message: alexaResponse,
        senderName: alexa.name}));

Reply:
Done.